editoast/src/views/timetable/stdcm/request.rs

```diff
@@ -297,6 +303,72 @@ impl Request {
             .await?;
         Ok(Some(towed_rolling_stock))
     }
+
+    pub(super) fn validate_consist(
+        &self,
+        traction_engine: &RollingStock,
+        towed_rolling_stock: &Option<TowedRollingStock>,
+    ) -> Result<()> {
+        self.validate_consist_mass(traction_engine, towed_rolling_stock)?;
+        self.validate_consist_length(traction_engine, towed_rolling_stock)?;
+        Ok(())
+    }
+
+    fn validate_consist_mass(
+        &self,
+        traction_engine: &RollingStock,
+        towed_rolling_stock: &Option<TowedRollingStock>,
+    ) -> Result<()> {
+        let max = Mass::new::<kilogram>(10000000.0);
+        let consist_mass = traction_engine.mass
+            + towed_rolling_stock
+                .as_ref()
+                .map(|t| t.mass)
+                .unwrap_or_default();
+        if let Some(request_total_mass) = self.total_mass {
+            if request_total_mass < consist_mass || request_total_mass > max {
+                return Err(StdcmError::RequestValidationFail {
+                    message: format!(
+                        "The total weight must be between {} and {}",
+                        &consist_mass.into_format_args(kilogram, DisplayStyle::Description),
+                        &max.into_format_args(kilogram, DisplayStyle::Description),
```

Comment:
To be explicit about the check and where the value comes from:
```suggestion
                        "The total mass must be greater than the sum of the rolling stock masses ({})",
                        &consist_mass.into_format_args(kilogram, DisplayStyle::Description),
```